cpp/libs/gst_nmos_plugins/README.md

```diff
@@ -0,0 +1,184 @@
+# NMOS GStreamer Plugins
+
+This repository contains GStreamer plugins that integrate AMWA NMOS functionality into your pipelines.  
+Specifically, it provides an **NMOS Audio Receiver** (`nmosaudioreceiver`), **NMOS Video Receiver** (`nmosvideoreceiver`), and **NMOS Sender** (`nmossender`) that can dynamically receive and send audio/video streams configured by the NMOS control panel.
+
+## Table of Contents
+
+## Table of Contents
+
+1. [Overview](#overview)
+2. [Building from Source](#building-from-source)
+3. [Installing the Plugin](#installing-the-plugin)
+   - [Installing via Script](#option-1-installing-via-script)
+   - [Installing Manually](#option-2-installing-manually)
+   - [Environment Variable](#option-3-environment-variable)
+   - [Verifying Installation](#verifying-the-installation)
+4. [Using the Plugin](#using-the-plugin)
+   - [Inspecting the Plugin](#inspecting-the-plugin)
+   - [Example Pipelines](#example-pipelines)
+   - [NMOS Interface](#nmos-interface)
+5. [Notes and Troubleshooting](#notes-and-troubleshooting)
+6. [License](#license)
+
+
+---
+
+## Overview
+
+The **NMOS plugins** communicate with an NMOS registry (using the AMWA [IS-04](https://specs.amwa.tv/is-04/) and [IS-05](https://specs.amwa.tv/is-05/) APIs) to automatically configure raw video/audio streams.
```

Comment:
The **NMOS plugins** **manage an NMOS controller** that communicates with an NMOS registry (using the AMWA [IS-04](https://specs.amwa.tv/is-04/) and [IS-05](https://specs.amwa.tv/is-05/) APIs) to automatically **register and control the NMOS resources that send/receive raw video/audio streams.**